.travis.yml

```diff
@@ -11,10 +11,11 @@ language: python
 python: "2.7"
 
 env:
-  - ANSIBLE_INSTALL_VERSION=1.9.4
   - ANSIBLE_INSTALL_VERSION=2.0.2.0
-  - ANSIBLE_INSTALL_VERSION=2.1.1.0
-  - ANSIBLE_INSTALL_VERSION=2.2.0.0
```

Comment:
You assume no one uses this version...

Reply:
I think it's a good idea to use the latest patch version of each major version.

Reply:
I don't we should test every patch, we'll end up testing sheds loads of versions. If you look at the recent updates these are the versions that we are testing. I agree that long term we should fine a way to centralise these versions, I open to suggestions on that.

Reply:
We don't run tests manually :)